smartmodule/examples/aggregate-init/transforms.yaml

```diff
@@ -0,0 +1,5 @@
+transforms:
+  - uses: infinyon/aggregate@0.1.0
+    with:
+      initial_value: "10"
+
```

Comment:
Can you also add README on how to run in the CLI as well?

Reply:
Done